docs/pages/access-controls/device-trust/device-management.mdx

```diff
@@ -0,0 +1,381 @@
+    ------------ ----- ------------- ------------------------------------
+    (=devicetrust.asset_tag=) windows not enrolled  (=devicetrust.device_id=)
+    ```
+  </TabItem>
+</Tabs>
+
+<Admonition type="warning" title="Device Role">
+For clusters created using Teleport v12 or newer, the builtin role editor has the necessary permissions to manage devices.
+If you Teleport cluster is older than v12, you will need to create and grant `device` role to manage device inventory.[link to below]
+</Admonition>
+
+## Generate enrollment token
+A registered device becomes a trusted device after it goes through the
+enrollment ceremony. The tie between the device inventory and enrollment
+is a device enrollment token. A device enrollment token needs to be created
+by a device admin and sent to the person performing the enrollment ceremony.
+
+The token ties the actual device being enrolled to the expected device in the
+inventory, adding an extra layer of protection.
+
+To create an enrollment token run the following command, where `--asset-tag` is
+the serial number of the device we want to enroll.
+
+```code
+$ tctl devices enroll --asset-tag="(=devicetrust.asset_tag=)"
+Run the command below on device "(=devicetrust.asset_tag=)" to enroll it:
+tsh device enroll --token=(=devicetrust.enroll_token=)
+```
+
+## Enrolling device
+
+To perform the enrollment ceremony, using the device specified above, type the
+command printed by `tctl devices enroll`:
+
+```code
+$ tsh device enroll --token=(=devicetrust.enroll_token=)
+Device "(=devicetrust.asset_tag=)"/macOS enrolled
+
+$ tsh logout
+$ tsh login --proxy=teleport.example.com --user=myuser # fetch new certificates
+Enter password for Teleport user myuser:
+Tap any security key
+Detected security key tap
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       myuser
+  Cluster:            teleport.example.com
+  Roles:              access, editor
+  Logins:             myuser
+  Kubernetes:         enabled
+  Valid until:        2023-06-23 02:47:05 -0300 -03 [valid for 12h0m0s]
+  Extensions:         teleport-device-asset-tag, teleport-device-credential-id, teleport-device-id
+```
+The presence of the `teleport-device-*` extensions shows that the device was
+successfully enrolled and authenticated. The device above is now a trusted device.
+
+<Admonition type="note" title="Windows">
+On Windows, the `tsh device enroll` command attempts to elevate to
+administrator privileges on the device. Privilege elevation is required to
+perform a TPM Credential Activation. This step is only required for
+enrollment and is not required on future device authentications.
+</Admonition>
+
+## Auto enrollment
+In a large organization, distributing enrollment token to user can be challenging.
+Teleport supports auto-enrollment, which if enabled, automatically enrolls user
+device during next Teleport login.
+
+Auto-enrollment allows `tsh` to automatically enroll devices already registered
```

Comment:
Is it Auto-enrollment or auto enrollment?

Reply:
Currently, its "auto-enrollment". Though in my heading, I have "auto enrollment". I will update it to auto-enrollment for consistency.